stix_shifter_modules/guardium/stix_transmission/utils/RestApiClient.py

```diff
@@ -5,17 +5,19 @@
 import urllib.parse
 import os
 import errno
+import uuid
 
 # This is a simple HTTP client that can be used to access the REST API
-CLIENT_CERT_NAME = "client_cert.pem"
-SERVER_CERT_NAME = "server_cert.pem"
 
 class RestApiClient:
     #cert_verify can be True -- do proper signed cert check, False -- skip all cert checks, or a Cert -- use the proper cleint side cert
     #mutual_auth is in the case the gateway is being used
     def __init__(self, host, port=None, cert=None, headers={}, url_modifier_function=None, cert_verify=True,
                  mutual_auth=False, sni=None):
         print("Guardium Version of RestApiClient ----")
+        uniqueFileHandle = uuid.uuid1()
```

Comment:
can you use uuid4() instead, to avoid potential collisions

Reply:
>>> import uuid
>>> uuid.uuid1()
UUID('f9c20d06-80dc-11ea-b54c-f0d5bffaeeb6')
>>> uuid.uuid4()
UUID('334da71b-7a35-4406-bfe9-6e161c91964e')
>>> str(uuid.uuid1())
'19ba7b02-80dd-11ea-b54c-f0d5bffaeeb6'
>>> len(str(uuid.uuid1()))
36
>>> len(str(uuid.uuid4()))
36
>>>

Reply:
uuid1 is based on mac address + timestamp. it's possible to be the same if generated at the exact same time on the same computer... very low chance.